src/main/java/mate/academy/security/FieldMatchValidator.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.security;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.apache.commons.beanutils.BeanUtils;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+    private String firstFieldName;
+    private String secondFieldName;
+    private String message;
+
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+        this.firstFieldName = constraintAnnotation.first();
+        this.secondFieldName = constraintAnnotation.second();
+        this.message = constraintAnnotation.message();
+    }
+
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext context) {
+        try {
+            final Object firstObj = BeanUtils.getProperty(value, firstFieldName);
```

Comment:
```suggestion
            final Object firstObj = new BeanWrapperImpl(value).getPropertyValue(this.firstFieldName);
```

the same for next field